src/index.js

```diff
@@ -21,8 +21,11 @@ function handlePlaceholder({ placeholder }) {
     return [placeholder];
 }
 
-function getSelectedItem({ items, value }) {
+function getSelectedItem({ items, key, value }) {
     let idx = items.findIndex((item) => {
+        if (item.key && key) {
```

Comment:
This may be written more succinctly as 
```
let index = items.findIndex(item => item.key && key ? isEqual(item.key, key) : isEqual(item.value, value));
```

Reply:
I personally avoid ternary operators, but we can have @lfkwtz weigh in